tests/test_wog.py

```diff
@@ -257,3 +257,34 @@ def test_wisps_of_the_old_gods():
 	for wisp in game.player1.field:
 		assert wisp.atk ==  wisp.health == 3
 		assert wisp.id == "OG_195c"
+
+def test_scaled_nightmare():
+	game = prepare_game()
+	scaled_nightmare = game.player1.give("OG_271")
+	scaled_nightmare.play()
+	game.end_turn(); game.end_turn()
+	assert scaled_nightmare.atk == 4
+	game.end_turn(); game.end_turn()
+	assert scaled_nightmare.atk == 8
+
+def test_scaled_nightmare_buff_ordering():  # To show that fireplace doesn't have blizzard's truly bizzare bugs.
```

Comment:
You should reference the hs-bugs entry directly (cf `test_dalaran_aspirant` for example)